processing/src/main/java/io/druid/segment/IndexMergerV9.java

```diff
@@ -752,75 +741,6 @@ private LongColumnSerializer setupTimeWriter(final IOPeon ioPeon) throws IOExcep
     return dimWriters;
```

Comment:
if we migrate indexing to use this class, I think we should port the timers and optimizations here too